eth/gaspricemonitor.go

```diff
@@ -24,9 +24,12 @@ type GasPriceMonitor struct {
 	polling         bool
 	pollingInterval time.Duration
 	cancel          context.CancelFunc
+	// pollingMu is a mutex that protects access to polling related
+	// fields
+	pollingMu sync.Mutex
 
-	// mu is a mutex that protects access to gasPrice
-	mu sync.Mutex
+	// gasPriceMu is a mutex that protects access to gasPrice
+	gasPriceMu sync.RWMutex
 	// gasPrice is the current gas price to be returned
```

Comment:
Same here, but maybe my screen is just too wide.

Reply:
Fixed in 2cec5ef